### PR TITLE
Add the submit response step

### PR DESCRIPTION
**Pull request by jmaspons**
In the author's response step there is no indication about submitting the comment to the bot

### CODE REVIEW

**Review by mpadge — APPROVED** [review]
Thanks @jmaspons for an important tweak!

**Comment by mpadge** [comment]
@maelle I guess the workflow is only failing because it's running from Joan's fork, and so missing the envvar, right? And that means it should work once it has been merged, right? If so, feel free to merge.

**Comment by mpadge** [comment]
@maelle Friendly reminder ...

**Comment by maelle** [comment]
I also need to add the same sentence to the Spanish file. Let me fetch the PR.

**Comment by maelle** [comment]
@jmaspons could you please review the Spanish text I added? :pray:

**Comment by maelle** [comment]
I am able to build the dev guide locally :smile_cat:

File: softwarereview_author.es.Rmd [hunk 1]
```diff
@@ -47,7 +47,7 @@ Esta guía condensa el proceso de revisión por pares desde el punto de vista de
...
+- Te pedimos que respondas a estos comentarios en un plazo máximo de 2 semanas desde la última revisión presentada, pero puedes actualizar tu paquete o responder en cualquier momento. Tu respuesta debe incluir un enlace a la actualización del archivo [*NEWS.md*](#news) de tu paquete. Aquí tienes [un ejemplo de respuesta](https://github.com/ropensci/software-review/issues/593#issuecomment-1714421144). Una vez que respondaste, [guardala en nuestros datos usando nuestro bot](/es/bot_cheatsheet.es#submit-response-to-reviewers). Fomentamos las conversaciones continuas entre quienes envían el paquete y quienes lo revisan. Consulta la [guía de revisión](#reviewerguide) para más detalles.
```
Comment (jmaspons) [inline softwarereview_author.es.Rmd:50]:
```suggestion
- Te pedimos que respondas a estos comentarios en un plazo máximo de 2 semanas desde la última revisión presentada, pero puedes actualizar tu paquete o responder en cualquier momento. Tu respuesta debe incluir un enlace a la actualización del archivo [*NEWS.md*](#news) de tu paquete. Aquí tienes [un ejemplo de respuesta](https://github.com/ropensci/software-review/issues/593#issuecomment-1714421144). Una vez hayas respondido, [enviala a nuestra base de datos usando nuestro bot](/es/bot_cheatsheet.es#submit-response-to-reviewers). Animamos a la continuación de conversaciones entre autores y revisores. Consulta la [guía de revisión](#reviewerguide) para más detalles.
```

Comment (jmaspons) [reply softwarereview_author.es.Rmd:50]: Also improving the next phrase. The translation is not 1:1, but it works (no "en nuestra base de datos" in the English version).

**Review by jmaspons — COMMENTED** [review]
Added changes in the spanish translation, @maelle

**Comment by maelle** [comment]
Thanks so much @jmaspons!!

**Pull request merged**